apps/coscrad-frontend/src/components/header/header.spec.tsx

```diff
@@ -25,23 +25,4 @@ describe('Header', () => {
 
         expect(screenRes).toBeTruthy();
     });
-
-    it('should load the subTitle from configurable content', () => {
```

Comment:
Why  is this removed? Did you stop using this content in the header? I think so, but please confirm.

Reply:
That's correct, subtitle was moved to the footer.  I moved the describe into the footer.spec.tsx as well.